app/views/lists/index.html.erb

```diff
@@ -0,0 +1,290 @@
+<%= render 'layouts/headersecond' %>
+
+<!-- BREADCRUMB -->
+<nav class="my-5">
+    <div class="container">
+    <div class="row">
+        <div class="col-12">
+
+        <!-- Breadcrumb -->
+        <ol class="breadcrumb mb-0 font-size-xs text-gray-400">
+            <li class="breadcrumb-item">
+            <a class="text-reset" href="<%= root_path %>"><%= t("layout.header.home")%></a>
```

Comment:
thay toàn bộ thẻ a thành link_to cho anh

Reply:
Dạ vâbg để em thử lại ạ. Tại em thấy nó không nhận css nên mới nghịch 😂😂